blocks/bricks/sequence_generators.py

```diff
@@ -405,6 +405,10 @@ def emit(self, readouts):
         return readouts
 
     @application
+    def cost(self, readouts, outputs):
+        return tensor.zeros_like(outputs)
```

Comment:
Can you please say in the documentation of this class that by default the `cost` method returns zero tensor?